Cargo.toml

```
@@ -3,7 +3,7 @@ members = ["cpp", "tools/*"]
 resolver = "2"
 
 [workspace.package]
-version = "0.8.6"
+version = "0.8.7"
```

This should be a breaking change.

```suggestion
version = "0.9.0"
```